ThirdPartyAdapters/unity/unity/src/main/java/com/google/ads/mediation/unity/UnityAdapter.java

```diff
@@ -282,34 +246,57 @@ public void requestBannerAd(
   @Override
   public void onDestroy() {
     mMediationInterstitialListener = null;
-    if (bannerAd == null) {
-      return;
+    if (bannerAd != null) {
+      bannerAd.onDestroy();
     }
-    bannerAd.onDestroy();
   }
 
   @Override
   public void onPause() {
-    if (bannerAd == null) {
-      return;
+    if (bannerAd != null) {
+      bannerAd.onPause();
     }
-    bannerAd.onPause();
   }
 
   @Override
   public void onResume() {
-    if (bannerAd == null) {
-      return;
+    if (bannerAd != null) {
+      bannerAd.onResume();
     }
-    bannerAd.onResume();
   }
 
   @Override
   public View getBannerView() {
-    if (bannerAd == null) {
-      return null;
+    if (bannerAd != null) {
+      return bannerAd.getBannerView();
+    }
+    return null;
+  }
+
+  private void sendInterstitialAdEvent(AdEvent adEvent) {
+    if (mMediationInterstitialListener == null) {
+      return;
+    }
+
+    switch (adEvent) {
+      case LOADED:
+        mMediationInterstitialListener.onAdLoaded(UnityAdapter.this);
```

Comment:
Redundant class qualifier (we are already in `UnityAdapter` so all these could by just `this` instead)